app/messaging/schema/submit-claim-schema.js

```diff
@@ -15,6 +15,7 @@ const submitClaimSchema = joi.object({
     urnResult: joi.string().required(),
     vetRcvs: joi.string().required(),
     detailsCorrect: joi.string().required(),
+    dateOfClaim: joi.string().optional(),
```

Comment:
This should probably be added to the new docs/asyncapi

Reply:
Added to docs/asyncapi within the claim service